webdriver/tests/bidi/browsing_context/print/background.py

```diff
@@ -46,8 +46,8 @@ async def test_background(
             "left": 0
         },
         page={
-            "width": px_to_cm(1),
-            "height": px_to_cm(1)
+            "width": pt_to_cm(1),
```

Comment:
Had to update this to use points instead of pixels for consistency with the minimum of 1x1 point, otherwise the dimensions end up being smaller, thus it would yield Invalid Argument which is not the purpose of this test.